src/app/routes/index.js

```diff
@@ -3,17 +3,20 @@ import GenericHandler from '../helpers/responses';
 import authRoutes from './auth/userRouters';
 
 const apiVersion = '/api/v1';
```

Comment:
Put this information into the **.env** file

Reply:
okay got it